lib/puppet/provider/cron/crontab.rb

```diff
@@ -1,7 +1,7 @@
 require_relative 'filetype'
 require 'puppet/provider/parsedfile'
 
-Puppet::Type.type(:cron).provide(:crontab, parent: Puppet::Provider::ParsedFile, default_target: ENV['USER'] || 'root') do
+Puppet::Type.type(:cron).provide(:crontab, parent: Puppet::Provider::ParsedFile, default_target: ENV['USER'] || 'root', raise_prefetch_errors: true) do
```

Comment:
If the module is installed on an older puppet agent whose parsedfile doesn't know about the `raise_prefetch_errors` option, will the option be ignored or will it fail?

Reply:
Ok, I ran the tests on a 6.0.0 agent and they passed. Here's the output:
```
Pending: (Failures listed here are expected and do not affect your suite's status)

  1) when Puppet cannot read a crontab file should not overwrite it on d7pg5p1blweghqf.delivery.puppetlabs.net
     # Skipping this test since we are on an older agent that does not have the PUP-9217 changes
     # ./spec/acceptance/tests/resource/cron/should_not_overwrite_crontab_file_on_file_read_error_spec.rb:47

  2) when Puppet cannot read a crontab file should only fail the associated resources on d7pg5p1blweghqf.delivery.puppetlabs.net
     # Skipping this test since we are on an older agent that does not have the PUP-9217 changes
     # ./spec/acceptance/tests/resource/cron/should_only_fail_associated_resources_on_file_read_error_spec.rb:45


Finished in 6 minutes 47 seconds (files took 19.6 seconds to load)
12 examples, 0 failures, 2 pending
```

To double check, I found https://github.com/puppetlabs/puppet/blob/master/lib/puppet/util/classgen.rb#L172-L177 from https://github.com/puppetlabs/puppet/blob/master/lib/puppet/type.rb#L1881 which confirms that the option's ignored because older agents do not have `raise_prefetch_errors=` defined.